.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore =
+	W503,		# line break before binary operator (follow PEP8 style guide)
+	E402,		# module level import not at top of file (because gi requires to query for versions before importing)
```

Comment:
In this particular case it's not pleasant to add to all the affected lines, since I can't move these imports to above the `gi.require_versions(...)`:

```py
gi.require_versions(...)

from gi.repository import Gio, Gtk, GtkClutter, Gdk, Gst, Clutter # noqa: E402

from komorebi.preferences_window import PreferencesWindow # noqa: E402
from komorebi.settings import ConfigKeys, Settings # noqa: E402
import komorebi.utilities # noqa: E402

from komorebi.screen import Screen # noqa: E402
...
```

But I found a [`per-file-ignores`](https://flake8.pycqa.org/en/latest/user/options.html?highlight=per-file-ignores#cmdoption-flake8-per-file-ignores) rule which makes this simpler.